grpc-server/src/main/java/com/netflix/conductor/grpc/server/service/HealthServiceImpl.java

```diff
@@ -1,17 +1,17 @@
 package com.netflix.conductor.grpc.server.service;
 
+import com.netflix.conductor.annotations.Audit;
 import com.netflix.runtime.health.api.HealthCheckAggregator;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import javax.inject.Inject;
-
 import io.grpc.health.v1.HealthCheckRequest;
 import io.grpc.health.v1.HealthCheckResponse;
 import io.grpc.health.v1.HealthGrpc;
 import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
 
+@Audit
```

Comment:
Audit is typically for logging user access to resources. Is it necessary to audit health check?

Reply:
hey sorry, I accidentally shared this PR with you, I removed all the reviewers yesterday but somehow missed you :) this PR is not quite ready yet...